libs/core/src/lib/object-marker/object-marker.component.ts

```diff
@@ -20,7 +20,8 @@ import { CssClassBuilder } from '@fundamental-ngx/core/utils';
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
-        '[attr.tabindex]': 'clickable ? 0 : -1'
+        '[attr.tabindex]': '0',
```

Comment:
Shouldn't we keep tabindex condition?

Reply:
As per the accessibility  requirement all the object marker label should be read on navigation so removed tabindex condition.